workflows/rnaseq-ref-index/build-index.nf

```diff
@@ -2,9 +2,11 @@
 nextflow.enable.dsl=2
 
 // basic parameters
-params.ref_dir = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100'
+params.ref_dir = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-103'
 params.cdna = 'fasta/Homo_sapiens.GRCh38.cdna.all.fa.gz'
-params.txome = 'fasta/Homo_sapiens.GRCh38.txome.fa.gz'
+params.spliced_txome = 'fasta/Homo_sapiens.GRCh38.103.spliced.txome.fa.gz'
+params.spliced_intron_txome = 'fasta/Homo_sapiens.GRCh38.103.spliced_intron.txome.fa.gz'
+params.ensembl_txome = 'fasta/Homo_sapiens.GRCh38.txome.fa.gz'
```

Comment:
This is probably fine for now, but if we find we are changing the inputs like this too often, we might want to just make the workflow more flexible, and allow specifying a list of transcriptome files and a parallel list of ids, rather than hard coding the IDs for each kind of transcriptome

Reply:
Yes definitely agreed. Maybe if we start changing things around more we can alter it to just have a fasta input and corresponding name like you said. Perhaps something to change if we try the toplevel.fa...